kotlin/src/main/kotlin/com/linecorp/armeria/client/kotlin/CoroutineRestClient.kt

```diff
@@ -29,14 +29,14 @@ import kotlinx.coroutines.future.await
  *
  * @see JacksonObjectMapperProvider
  */
-suspend inline fun <reified T> RestClientPreparation.execute(): ResponseEntity<T> {
+suspend inline fun <reified T : Any> RestClientPreparation.execute(): ResponseEntity<T> {
```

Comment:
Note: https://stackoverflow.com/questions/63535390/how-is-any-upper-bound-in-kotlin-different-from-the-default-no-bounds
I didn't know that `<T>` bound is the same as `<T : Any?>`. TIL 🙇‍♂️